client-registry-api/src/main/java/uk/gov/di/authentication/clientregistry/lambda/ClientRegistrationHandler.java

```diff
@@ -67,16 +69,12 @@ public APIGatewayProxyResponseEvent handleRequest(
     public APIGatewayProxyResponseEvent clientRegistrationRequestHandler(
             APIGatewayProxyRequestEvent input, Context context) {
         String ipAddress = IpAddressHelper.extractIpAddress(input);
-        auditService.submitAuditEvent(
-                REGISTER_CLIENT_REQUEST_RECEIVED,
-                AuditService.UNKNOWN,
-                context.getAwsRequestId(),
-                AuditService.UNKNOWN,
-                AuditService.UNKNOWN,
-                AuditService.UNKNOWN,
-                ipAddress,
-                AuditService.UNKNOWN,
-                AuditService.UNKNOWN);
+
+        var user =
+                TxmaAuditUser.user()
+                        .withSessionId(context.getAwsRequestId())
```

Comment:
Is this correct? It looks like this was originally added explicitly as the request ID: https://github.com/govuk-one-login/authentication-api/commit/7ef78e33315866863e07f203d2bf97e4e48f589c#diff-a7dd2094ffa7958a40dfc6b333bf0d588539d0eccc5bf89d37efc6cdc13fbec7

Reply:
Discussed on Slack, while this is not a regression, it doesn't really make sense as a value - would probably be a better fit for `transactionId`